lib/vector-buffers/src/variants/disk_v2/ledger.rs

```diff
@@ -705,7 +705,7 @@ where
 
     #[must_use]
     pub(super) fn spawn_finalizer(self: Arc<Self>) -> OrderedFinalizer<u64> {
-        let (finalizer, mut stream) = OrderedFinalizer::new(ShutdownSignal::noop());
+        let (finalizer, mut stream) = OrderedFinalizer::new(None);
```

Comment:
Should this not be `Some` here too, at least for now? The description says only the `file` source is being modified.

Reply:
Oh, yes I should have mentioned this case too. If I understand correctly, `ShutdownSignal::noop()` gives back a `ShutdownSignal` that is not attached to the global shutdown trigger, and can't be triggered externally. In this case, I believe this "noop" `ShutdownSignal` should be functionally equivalent to passing `None`, since it can't be signaled until the finalizer stream that owns it is dropped?

Reply:
FWIW, after adding a unit test for the file source, I ran those tests locally using `Some(ShutdownSignal::noop())`  in the file source ack stream, and that also fixes the bug - evidence that it is indeed equivalent to passing `None`.

That said, if there is a concern with including this change here I'm happy to revert this line for now.